### DISCUSSION TITLE
Conflict with Datepicker using Checkout Field Editor for WooCommerce (Pro)

### DISCUSSION THREAD

**Discussion by Dreamabstract** [bootscore/bootscore#451] [General]
We've discovered a conflict with the jquery datepicker from Checkout Field Editor for WooCommerce (Pro) 

Example here if you add a product and go to the checkout https://stagingabstract.com/bootscore/product/test-product/

Has anyone else encountered this and know of a way to get them to work together.

**Comment by crftwrk** [comment]
Hello,

I see no datepicker in your checkout, is this the conflict you are meaning? First issue I see is that the quantity input buttons are twice in product page. Means that there seems a general conflict with Checkout Field Editor plugin.

<img width="270" alt="Bildschirmfoto 2023-04-15 um 12 12 32" src="https://user-images.githubusercontent.com/51531217/232207765-4fc91608-e523-4e33-aa7b-a914aed7d38e.png">

Try to do this:

- Disable maxmegamenu and all other plugins (except WooCommerce) and check if product page works.
- Enable Checkout Field Editor plugin and check product page again.

If it's definitive done by Checkout Field Editor, then we have to investigate further how this plugin works and why qty buttons are twice.

But one more question. Is the datepicker the only thing what you want to do with this plugin? If so, this can be made by a simple function.

**Comment by Dreamabstract** [comment]
Hi

I didn't spot that problem with the quantity, it appears that happens even when you disable the checkout fields plugin. The issue I was trying to resolve was if you add a product to the cart and then navigate to the checkout you will see that the date picker does not load when you click on the Delivery Date field. This problem is actually occurring on another website in production using a slightly older version of the theme. This version on the test site uses the latest version of the Bootscore theme. I've now disabled all plugins except WooCommerce, it's running the Bootcommerce child theme.

As they already use the checkout field plugin we were hoping to use it as it can allow us to easily set dates that deliveries are not available and a 2 day wait after ordering. I am aware that we could try this another way using ACF, perhaps there is a better option for Bootstrap using a different date picker?

Andy

**Reply by crftwrk** [reply]
> I didn't spot that problem with the quantity, it appears that happens even when you disable the checkout fields plugin.

I think it is not the checkout fields plugin, seems that this is done by the max mega menu. That's why I ask in previous comment to disable ALL plugins first and then enable them step by step until the devil is found ;-)

**Reply by Dreamabstract** [reply]
Fixed the quantity issue, it looks like there must of been an old theme file somwhere. I deleted all the theme folders and uploaded a fresh copy.

The issue with the date picker remains.

**Comment by justinkruit** [comment]
I see where you got the datepicker in the checkout (delivery date), but there doesn't seem to be a js error which you would expect in this case... I would need to investigate more why it isn't triggering.

**Reply by Dreamabstract** [reply]
We're updating an existing Woo site with a new custom theme and they had an old function to load a date picker with javascript and that also didn't load the date picker so assume it must be something blocking it somewhere.
